docs/guides/local-apt-repository.rst

```diff
@@ -19,14 +19,15 @@ A lot of other roles will compile from source  but that's time demanding and
 error prone. A backported version of Ruby 2.1.x will apt install in about 5
 seconds once you setup your local APT server once.
 
+.. note::
     Compile it once into a proper package and use it as many times as you want.
 
-It also future proofs your role because you wouldn't have to change anything
+It also makes your role future proof because you don't have to change anything
 once the next Debian version is officially released. From the role's point of view it's just
 installing an apt package using Ansible's :command:`apt` module. It does not care where
 the apt server is located.
 
-Pick a server
+Picking a host for the repository
 -------------
```

Comment:
The underline needs to match the header, otherwise it won't be parsed correctly by Sphinx.